scripts/release-app-interface.sh

```
@@ -232,6 +240,9 @@ initKokuUISHA()
 # Use gh in a non-interactive way -- see https://github.com/cli/cli/issues/1718
 mergeRequest()
 {
+  # tagRelease cds to koku-ui; the deploy branch lives in the app-interface clone.
+  cd $APP_INTERFACE_DIR
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)